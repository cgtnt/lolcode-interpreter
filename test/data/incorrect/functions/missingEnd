HAI

HOW IZ I FOO
VISIBLE "hi"

KTHXBYE
