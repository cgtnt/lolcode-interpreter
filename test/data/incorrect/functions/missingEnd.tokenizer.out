@1:T_BEGIN:HAI-@3:T_COMMAND_TERMINATOR:
-@3:T_FUNC_BEGIN:HOW IZ I-@3:T_T_IDENTIFIER:FOO-@4:T_COMMAND_TERMINATOR:
-@4:T_WRITE_STDOUT:VISIBLE-@4:T_T_STRING:"hi"-@6:T_COMMAND_TERMINATOR:
-@6:T_END:KTHXBYE-@7:T_COMMAND_TERMINATOR:
-@7:T_EOF:
